Vagrantfile

```diff
@@ -9,6 +12,7 @@ Vagrant.configure("2") do |config|
     vb.gui = false
     vb.linked_clone = true
     vb.memory = 2048
+    vb.customize ["modifyvm", :id, "--cableconnected1", "on"]
```

Comment:
This could use an explanation - I will have to read in the VirtualBox docs to figure out why this was needed. Maybe related to VM reboot?

Reply:
I was trying to bring up the machines on my stationary pc, and it didnt work. I found this post, added it and it worked. I can try and remove it when i get back home again to double check.

https://github.com/hashicorp/vagrant/issues/8056#issuecomment-267600935

Reply:
Seems due to a bug fixed in VirtualBox 5.1:
hashicorp/vagrant#7648